.github/actions/set-env-variables/action.yml

```diff
@@ -10,6 +10,6 @@ runs:
         # no prod yet
         echo "QUAY_CHARTS_ORGANIZATION_DEV=cilium-charts-dev" >> $GITHUB_ENV
         # renovate: datasource=github-releases depName=cilium/cilium-cli
-        CILIUM_CLI_VERSION="v0.15.23"
+        CILIUM_CLI_VERSION="v0.16.0"
```

Comment:
The readme states this isn't compatible. Is the readme incorrect..?

https://github.com/cilium/cilium-cli?tab=readme-ov-file#releases

Reply:
yeah this is what i wanted to check with @sayboras. let's see what he says

Reply:
It might not work well if gateway api is enabled i.e. sysdump on crds, I think we can live with that.